src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -143,12 +144,26 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     // this removes any HTML from the message content
     String cleanedMessageContent = Jsoup.clean(messageContent, Whitelist.none());
 
+    /* use MarkdownProcessor to parse any markdown if the message
+     * contains markdown
+     */
+    MarkdownProcessor mark_it = new MarkdownProcessor();
+    String parsedMessageContent = mark_it.markdown(cleanedMessageContent);
+    // Remove <p> and </p> that markdown processor adds to string
+    int i = parsedMessageContent.length();
+    /* deleted 5th character from the end b/c markdown processor
```

Comment:
Can you provide examples of before/after - i.e. sample input markdown, what the processor will return, and what you're trimming off?